salt/utils/network.py

```diff
@@ -206,6 +207,7 @@ def is_ip(ip):
     return is_ipv4(ip) or is_ipv6(ip)
 
 
+@jinja_filter('is_ipv4')
```

Comment:
Sure, those are just there as working examples at the moment. Planning to remove those commits after the discussion. The only filters I'd include in this PR are those that already exist.